dashboard/v1.1/src/layouts/BaseLayout.tsx

```diff
@@ -120,15 +143,19 @@ export default function BaseLayout(props: any): ReactElement {
             >
               Bench Routes
             </Typography>
-            <Tooltip title="Dark Mode">
-              <Switch
-                checked={props.darkMode}
-                onChange={handleToggleDarkMode}
-                color="default"
-                name="checkedB"
-                inputProps={{ 'aria-label': 'primary checkbox' }}
-              />
-            </Tooltip>
+            <Switch
+              checked={props.darkMode}
+              onChange={handleToggleDarkMode}
+              offColor="#145D97"
+              onColor="#303030"
+              name="checkedB"
+              uncheckedIcon={
+                <Brightness7Sharp className={sunIconClasses.root} />
+              }
+              checkedIcon={
+                <Brightness2Sharp className={moonIconClasses.root} />
```

Comment:
@iamkryptonite This looks good. Only a small change is there any chance we can mirror image the moon icon or use something respective to that as the present icon looks a bit weird.

Reply:
Sure, I'll change the icon so that it looks more like a moon.

Reply:
@muskankhedia should I make the moon icon a little tilted. I think that might look better.

Reply:
@iamkryptonite yes you can